source/clear-linux/tutorials/hadoop.rst

```diff
@@ -0,0 +1,177 @@
+
+   .. code-block:: xml
+
+      <?xml version="1.0" encoding="UTF-8"?>
+      <?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+      <configuration>
+      <property>
+      <name>fs.default.name</name>
+      <value>hdfs://localhost:9000</value>
+      </property>
+      </configuration>
+
+#. Edit :file:`/etc/hadoop/hdfs-site.xml`. This file configures the
+   :abbr:`HDFS (Hadoop Distributed File System)` daemons. This includes
+   things like the list of permitted and excluded data nodes or the size of
+   the blocks. In this example, we are setting the number of block
+   replication to 1 from the default of 3 as follows:
+
+   .. code-block:: xml
+      :emphasize-lines: 6
+
+      <?xml version="1.0" encoding="UTF-8"?>
+      <?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+      <configuration>
+      <property>
+      <name>dfs.replication</name>
+      <value>1</value>
+      </property>
+      <property>
+      <name>dfs.permission</name>
+      <value>false</value>
+      </property>
+      </configuration>
+
+#. Edit :file:`/etc/hadoop/mapred-site.xml`. This file configures all daemons
+   related to MapReduce: `JobTracker` and `TaskTrackers`. With MapReduce,
+   Hadoop can process big amounts of data in multiple systems. In our
+   example, we set :abbr:`YARN (Yet Another Resource Manager)` as our runtime
+   framework for executing MapReduce jobs as follows:
+
+   .. code-block:: xml
+      :emphasize-lines: 5,6
+
+      <?xml version="1.0" encoding="UTF-8"?>
+      <?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+      <configuration>
+      <property>
+      <name>mapreduce.framework.name</name>
+      <value>yarn</value>
+      </property>
+      </configuration>
+
+#. Edit :file:`/etc/hadoop/yarn-site.xml`. This file configures all daemons
+   related to YARN: `ResourceManager` and `NodeManager`. In our example, we
+   implement the `mapreduce_shuffle` service, which is the default as follows:
+
+   .. code-block:: xml
+      :emphasize-lines: 4,5,8,9
+
+      <?xml version="1.0"?>
+      <configuration>
+      <property>
+      <name>yarn.nodemanager.aux-services</name>
+      <value>mapreduce_shuffle</value>
+      </property>
+      <property>
+      <name>yarn.nodemanager.auxservices.mapreduce.shuffle.class</name>
+      <value>org.apache.hadoop.mapred.ShuffleHandler</value>
+      </property>
+      </configuration>
+
+Run the Hadoop daemons
+**********************
+
+With all the configuration files properly edited, we are ready to start the
+daemons.
+
+When we format `NameNode`, it formats the meta-data related to data-nodes.
+Thus, all the information on the data nodes is lost and the nodes can be
+reused for new data.
+
+#. Format `NameNode` with the following command:
+
+   .. code-block:: bash
+
+      hdfs namenode -format
+
+#. Start the DFS daemons `NameNode` and `DataNodes` with the following command:
+
+   .. code-block:: bash
+
+      start-dfs.sh
```

Comment:
Running this command outputs the following (which is unspecified):
```
# start-dfs.sh 
Starting namenodes on [localhost]
The authenticity of host 'localhost (::1)' can't be established.
ECDSA key fingerprint is SHA256:97e+7TnomsS9W7GjFPjzY75HGBp+f1y6sA+ZFcOPIPU.
Are you sure you want to continue connecting (yes/no)?
```
maybe there's some pre-work to run this to avoid this thing?
That or document how to proceed here, please 👍

Reply:
Done